src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -363,6 +363,21 @@
              Link="Common\Interop\FreeBSD\Interop.Process.GetProcInfo.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'illumos' or '$(TargetPlatformIdentifier)' == 'solaris'">
+    <Compile Include="System\Diagnostics\Process.BSD.cs" />
+    <Compile Include="System\Diagnostics\Process.SunOS.cs" />
+    <Compile Include="System\Diagnostics\ProcessManager.SunOS.cs" />
+    <Compile Include="System\Diagnostics\ProcessThread.SunOS.cs" />
+    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.TimeSpec.cs"
+             Link="Common\Interop\Unix\System.Native\Interop.TimeSpec.cs" />
```

Comment:
That was part of a couple prerequisite fixes from @AustinWise that I was going to keep in a separate PR.
Should I just keep with this?  I guess I will, as separate commits for now...

Reply:
I think if we put all patches related to this library port on sunos-likes in this PR, it will help the maintainers to see the full scope and give us better guidance during their review.